platform/binance/api.go

```diff
@@ -116,3 +118,54 @@ func NormalizeTxs(srcTxs []Tx, token string) (txs []blockatlas.Tx) {
 	}
 	return
 }
+
+func (p *Platform) GetTokenListByAddress(address string) (blockatlas.TokenPage, error) {
+	account, err := p.client.GetAccountMetadata(address)
+	if err != nil {
+		return nil, err
+	}
+	tokens, err := p.client.GetTokens()
```

Comment:
Can you call `account` and `tokens` simultaneously ?
Ideally would be nice to store tokens in DB to avoid call to RPC every time some once requests a token list.

Reply:
I don't see a database library in the project, but I like your suggestion. I don't want to put a new library to support the Binance tokens, and I followed the Blockatlas doc:

["A library specific to your chain is not a necessary dependency."](https://developer.trustwallet.com/blockatlas/newblockchain)